xds/internal/balancer/clusterimpl/balancer_test.go

```diff
@@ -356,7 +362,10 @@ func (s) TestDropCircuitBreaking(t *testing.T) {
 		Service:    testServiceName,
 		TotalDrops: uint64(maxRequest),
 		LocalityStats: map[string]load.LocalityData{
-			assertString(xdsinternal.LocalityID{}.ToString): {RequestStats: load.RequestData{Succeeded: uint64(rpcCount - maxRequest + 50)}},
+			assertString(xdsinternal.LocalityID{}.ToString): {RequestStats: load.RequestData{
+				Succeeded: uint64(rpcCount - maxRequest + 50),
+				Issued:    uint64(rpcCount - maxRequest + 50),
```

Comment:
Are there no tests that include failed calls?  It would be nice to have at least one test case where succeeded != issued.

Reply:
Changed a couple of test cases to have both errors and successful RPCs.